poc/dst.py

```diff
@@ -17,22 +17,24 @@
 USAGE_JOINT_RAND_SEED: int = 3
 USAGE_JOINT_RAND_PART: int = 4
 USAGE_JOINT_RAND: int = 5
+USAGE_ONEHOT_CHECK: int = 6
+USAGE_PAYLOAD_CHECK: int = 7
+USAGE_EVAL_PROOF: int = 8
 
-# Vidpf
-USAGE_ONEHOT_PROOF_INIT: int = 6
-USAGE_ONEHOT_PROOF_HASH: int = 7
-USAGE_NODE_PROOF: int = 8
-USAGE_EVAL_PROOF: int = 9
+# VIDPF
+USAGE_NODE_PROOF: int = 9
 USAGE_EXTEND: int = 10
 USAGE_CONVERT: int = 11
-USAGE_PAYLOAD_CHECK: int = 12
 
 
 def dst(ctx: bytes, usage: int) -> bytes:
+    assert usage in range(256)
```

Comment:
could you explain this?

Reply:
`usage` should be a byte: I wanted to make sure the caller is actually passing a byte.

Reply:
Shouldn't we check if it's <= than the biggest value we have (`11` here)?